### DISCUSSION TITLE
Intégration userpilot

### DISCUSSION THREAD

**Discussion by OlivierVS31** [AmauriC/tarteaucitron.js#872] [Ideas]
Nous utilisons la solution userpilot pour aider et accompagner nos utilisateurs https://userpilot.com/ serait-il possible de l'avoir dans liste des solutions par défaut ?

**Comment by AmauriC** [comment]
Bien sûr mais il me faut:

- le marqueur HTML du service
- la liste des cookies installé par ce service
- l'URL de la politique de confidentialité
- des infos sur les éventuelles options

**Reply by OlivierVS31** [reply]
Bonjour,

Le lien vers le code d'intégration > [Userpilot Script](https://docs.userpilot.com/article/21-install-userpilot-on-your-web-app)

Et les infos globales : [Politique sécurité/cookies userpilot](https://userpilot.com/security)

J'essaye d'avoir plus de détail suite à ta demande auprès de la solution.

**Reply by AmauriC** [reply]
Merci, suite sur https://github.com/AmauriC/tarteaucitron.js/issues/892
